cool-core/src/main/java/com/nus/cool/core/io/readstore/ChunkRS.java

```diff
@@ -75,24 +75,25 @@ public int records() {
   public void readFrom(ByteBuffer buffer) {
     // Get chunkType
     ChunkType chunkType = ChunkType.fromInteger(buffer.get());
-      if (chunkType != ChunkType.DATA) {
-          throw new IllegalStateException("Expect DataChunk, but reads: " + chunkType);
-      }
+    if (chunkType != ChunkType.DATA) {
+      throw new IllegalStateException("Expect DataChunk, but reads: " + chunkType);
+    }
 
     // Get #records
     this.records = buffer.getInt();
     // Get #fields
     int fields = buffer.getInt();
     // Get field offsets
     int[] fieldOffsets = new int[fields];
```

Comment:
Not sure why we had this before. Just removed them instead of leaving them as comments?

Reply:
This is automatically modified by Javalint.
In the old version, there are two spaces.